lib/Gedmo/Tree/TreeListener.php

```diff
@@ -56,6 +56,18 @@ public function getSubscribedEvents()
         );
     }
 
+    /**
+     * @param string   $name
+     * @param Strategy $instance
+     * @return self
+     */
+    public function addStrategyInstance($name, $instance)
```

Comment:
What do you think about type-hinting `Gedmo\Tree\Strategy` here for `$instance`?